include/deal.II/particles/partitioner.h

```diff
@@ -97,6 +97,18 @@ namespace Particles
       std::vector<typename std::multimap<internal::LevelInd,
                                          Particle<dim, spacedim>>::iterator>
         ghost_particles_iterators;
+
+      /**
+       * Vector of char that is used to organize the particle information to be
+       * sent to other processors
+       */
+      std::vector<char> send_data;
+
+      /**
+       * Vector of char that is used to organize the particle information to
+       * received from other processors
+       */
+      std::vector<char> recv_data;
```

Comment:
I would suggest to adjust the comment similar to:

https://github.com/dealii/dealii/blob/79b124c53968490d700fa798d0a794756bc5f06a/include/deal.II/lac/la_parallel_vector.h#L1334-L1339

Reply:
Done in latest commit :). Thanks for the heads up !